src/providers/TransactionBuilderContextProvider.tsx

```diff
@@ -781,6 +819,49 @@ const TransactionBuilderContextProvider = ({
     setShowTransactionBlockSelect(false);
   };
 
+  const connectionCheck = async () => {
+    if (sdk && connect) {
+      setIsWalletConnecting(true);
+      try {
+        const address = await connect();
```

Comment:
address not used, no need to assign
```suggestion
        await connect();
```